README.md

```diff
@@ -292,7 +292,19 @@ The maximum number of retries to attempt. Default: 5.
 - Type: `function`
 - *Optional*
 
-Custom function returning a boolean inidicating whether or not to retry an error.
+Custom function returning a boolean indicating whether or not to retry an error.
+
+
+##### config.chunkSize
+
+- Type: `number`
+- *Optional*
+
+Enables [Multiple chunk upload](https://cloud.google.com/storage/docs/performing-resumable-uploads#chunked-upload) mode and sets each request size to this amount.
+
+This only makes sense to use for larger files. The chunk size should be a multiple of 256 KiB (256 x 1024 bytes), unless it's the last chunk that completes the upload. Larger chunk sizes typically make uploads more efficient. We recommend using at least 8 MiB for the chunk size.
```

Comment:
The comment about the last chunk is not really relevant here since they are setting this param once before the upload is initialized.

Reply:
Thanks, updated